config.ini.sample

```diff
@@ -22,6 +23,7 @@ stt_transcriptions_file=stt_transcriptions.csv
 audio_file_folder=.
 
 [ErrorRateOutput]
+report_directory=experiments
```

Comment:
I don't want to add configuration variables used only by a 'sample' (the experiment).  I think you could rename to `output_directory` and then make sure that is consistently used in the other programs.  Alternately, the report directory could be include in the output file names, though then we need to verify the directory is created first.

If this variable is truly only used in the 'sample' (the experiment), you could hardcode its value in that file.

Reply:
Changed to output_directory and made sure that all scripts use the value appropriately.